wire-ios-data-model/Source/Proteus/ProteusSessionID.swift

```diff
@@ -56,12 +56,8 @@ public struct ProteusSessionID: Hashable, Equatable {
         self.init(userID: "", clientID: clientID)
     }
 
-    public var hashValue: Int {
-        return self.rawValue.hashValue
-    }
-
     public func hash(into hasher: inout Hasher) {
```

Comment:
Can't this method be removed?

Reply:
Nope! We need to use that. I wanted to get read of the warning aka remove the property.